crates/typed-store/src/rocks/mod.rs

```diff
@@ -2100,7 +2100,7 @@ pub struct ReadWriteOptions {
 impl ReadWriteOptions {
     pub fn readopts(&self) -> ReadOptions {
         let mut readopts = ReadOptions::default();
-        readopts.set_ignore_range_deletions(self.ignore_range_deletions);
+        readopts.set_ignore_range_deletions(false);
```

Comment:
it needs to be set back to `readopts.set_ignore_range_deletions(self.ignore_range_deletions);`

Reply:
Done